### DISCUSSION TITLE
colormap questions

### DISCUSSION THREAD

**Discussion by gagecarto** [developmentseed/titiler#510] [Q&A]
I am working through a solution for a custom colormap. My datasets is scaled 0-58. I rescale in my titiler requests and then apply a viridis color ramp. This works great. 

http://myurl/WebMercatorQuad/9/105/186@1x?
url=myCogUrl
&bidx=1
&nodata=999
&rescale=0,58
&colormap_name=viridis

**Questions...** 
Can I easily reverse/flip the rio-tiler color ramp?

If wanted to apply my own color ramp made up of 3 colors ['#f0ff00', '#ffb000', '#ff0000'], can I do that directly in my tile request urls?

**Comment by dvd3v** [comment]
Yes this is possible if you add a custom Colormap dependency. There is an example in the documentation: https://devseed.com/titiler/examples/code/tiler_with_custom_colormap/

**Reply by gagecarto** [reply]
Thanks... How about reversing an existing color ramp?

**Reply by vincentsarago** [reply]
@gagecarto 

> Can I easily reverse/flip the rio-tiler color ramp?

most of the Colormap are provided by `matplotlib` and have both regular and reverse. you can try using `viridis_r`
